powerflex-ansible/site.yaml

```diff
@@ -29,44 +29,45 @@
   gather_facts: no
   become: true
   roles:
-   - powerflex-mdm
+   - { role: "powerflex-mdm", when: "'EMC-ScaleIO-mdm' not in ansible_facts.packages" }
   tags: powerflex-mdm
 
 - hosts: tbs
   name: "Install PowerFlex TB"
   gather_facts: no
   become: true
   roles:
-   - powerflex-tb
+   - { role: "powerflex-tb", when: "'EMC-ScaleIO-mdm' not in ansible_facts.packages" }
```

Comment:
should this tb not mdm

Reply:
No. The reason is because a tie-breaker is a type of mdm, hence the statement.